### PR TITLE
fix: [docs] Update README.md, added an installation step

### PR DESCRIPTION
**Pull request by iamthuya**
[docs] Added an installation step

### CODE REVIEW

**Comment by SmartManoj** [comment]
Fixes #8

**Review by jscud — APPROVED** [review]
Thanks!

**Comment by jscud** [comment]
Would it be possible to amend to meet the conventionalcommits requirements?

File: README.md [hunk 1]
```diff
@@ -7,6 +7,11 @@
 
 -----
 
+## Installation
+```cmd
```
Comment (jscud) [inline README.md:11]: Other ``` blocks have a space before the language, so better to write as ``` cmd.

**Review by jscud — COMMENTED** [review]
Could the commit also be marked with fix: to satisfy conventionalcommits tooling?

File: README.md (hunk 1 above)
Comment (jscud) [reply README.md:11]: Ha, it interpreted the markdown in my comment. I mean there should be a space after the triple backticks (`), before cmd.

Comment (SmartManoj) [reply README.md:11]: Space not mentioned [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting)

**Review by jscud — APPROVED** [review]

**Comment by jscud** [comment]
We'll be including this in the next release of the SDK, so marking this as closed.

**Pull request closed without merging**